backend/src/auth/controllers/auth-controller.ts

```diff
@@ -0,0 +1,22 @@
+import { Body, Controller, Post, UseGuards } from "@nestjs/common";
+import { AuthService } from "auth/services/auth.service";
+import { UserCredentialsDto } from "users/model/dto/userCredentials.dto";
+import { LocalAuthGuard } from "../guards/local-auth.guard";
+
+@Controller("auth")
+class AuthController {
+	constructor(private authService: AuthService) {}
+
+	@Post("register")
+	async register(@Body() loginRegisterUserDto: UserCredentialsDto) {
+		return this.authService.register(loginRegisterUserDto);
+	}
+
+	@UseGuards(LocalAuthGuard)
+	@Post("login")
+	async login(@Body() { username, password }: UserCredentialsDto) {
+		return this.authService.login(username, password);
```

Comment:
Just pass userCredentialsDto object to the service.

Reply:
That's rigth, i will do it :)